Documentation/AuthConflicsBetweenSDKs.md

```diff
@@ -0,0 +1,47 @@
+# Bridging gap between old and new SDKs
```

Comment:
you have 3 files in the PR:
```
1. Documentation/AuthConflicsBetweenSDKs.md
2. Documentation/AuthConflictsBetweenSDKs.md
3. Documentation/README.md
```

What is the difference between 1 and 2, except file name typo?